tests/containers/docker_runc.pm

```diff
@@ -31,7 +31,7 @@ sub run {
     my ($running_version, $sp, $host_distri) = get_os_release;
 
     my @runtimes = ();
-    push @runtimes, "docker-runc" if (is_sle("<16") || is_leap("<16.0"));
+    push @runtimes, "docker-runc" if (is_leap("<16.0") || is_sle("12-SP4"));
```

Comment:
```suggestion
    push @runtimes, "docker-runc" if (is_leap("<16.0") || is_sle("<12-SP5"));
```

Reply:
@Vogtinator is it expected to work for 12-sp3? https://openqa.suse.de/tests/5922340

Reply:
We have 12 SP3, SP4 and SP5, but of those only SP4 didn't get the new `runc`? That seems wrong

Reply:
So for ref: [sle12-sp4](https://openqa.suse.de/admin/products) was missing `contm` from `MRU_ADDONS`. Without it, 
the test was using the older docker version(`docker-19.03.15_ce`) which didnt cause conflicts. Said that, relocating the variable and filter out all the sle versions from running the docker-runc. Thanks @Vogtinator